index.html

```diff
@@ -0,0 +1,171 @@
+
+  <link rel="stylesheet" href="general.css">
+  <link rel="stylesheet" href="index.css">
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+    integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <title>My capstone project</title>
+</head>
+
+<body>
+
+  <main class="main-page">
+    <section class="first-page">
+      <header class="toolbar-logout">
+        <nav>
+          <div class="logout">
+            <a class="fb" href=""><i class="fab fa-facebook light-grey"></i></a>
+            <a class="twit" href=""><i class="fab fa-twitter light-grey"></i></a>
+            <a class="logout-b" href="">Logout</a>
+            <a class="mypage" href="">Mypage</a>
+            <a class="tigrigna" href="">ትግርኛ</a>
+          </div>
+          <div class="toolbar">
+            <div class="image-a"><img src="assets/logo.png" alt="logo"></div>
+            <ul>
+              <li><a href="./about.html">About</a></li>
+              <li>Programs</li>
+              <li>Join</li>
+              <li>Partners</li>
+              <li>news</li>
+              <li>CC compaign</li>
+            </ul>
+          </div>
+          <div class="humberger" onclick="openNav()">
+            <i class='fa fa-bars menu-m'></i>
+          </div>
+          <div id="myNav" class="m-menu">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+            <div class="m-menu-content">
+
+              <a class="abt" href="./about.html" onclick="closeNav()">About</a>
+              <a hre="#speakers" onclick="closeNav()">Speakers</a>
+              <a href="#partners" onclick="closeNav()">Partners</a>
+              <a href="#" onclick="closeNav()">Join the Program</a>
+              <a href="#section-044" onclick="closeNav()">Contact</a>
+              <a href="#main-program" onclick="closeNav()">Main Program</a>
+
+
+            </div>
+          </div>
+        </nav>
+
+      </header>
+      <div class="agenda">
+        <div class="headers">
+          <h2 class="hello special-color">"Hello! Sharing world"</h2>
+          <h1 class="main-topic special-color">Group Engineering ENTREPRENEURSHIP & TECHNOLOGY CONFERENCE 2023
+          </h1>
+        </div>
+        <p class="introductory">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et, illo! Rem cupiditate
+          nemo minima voluptate laboriosam deserunt, dignissimos quia incidunt id pariatur odit laborum sunt
+          reprehenderit labore blanditiis temporibus modi.
+        </p>
+        <div class="info">
+          <p class="date">
+            2023.06.28(MON) ~ 30(WED)
+          </p>
+          <p class="adress">@Hall of Mekelle institute of technology, Mekelle, Tigrai
+          </p>
```

Comment:
- [ ] Kindly, you need to stick to the original template as much as possible. the spacing(margin, padding) of your design is supposed to be similar to the original template. as suggested by the previous reviewer, kindly make sure these elements are well aligned as they are in the original template.
`Hint: You probably set the:`
```css
text-align: center; whereas it's not supposed to be the case, Kindly consider looking into that.
```
### Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/227734149-4ca457a0-2b83-4eac-87a6-13b973304624.png)
### Original template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/227734399-c9c6f8a3-9b0f-4426-8aac-03844e665a0b.png)